### DISCUSSION TITLE
Using Brunch to update Chromebooks past AUE date

### DISCUSSION THREAD

**Discussion by jfield9734** [sebanc/brunch#712] [General]
I just wanted to say that the Brunch solution not only works great for PC's, but also on Chromebooks that are past their AUE date and no longer receive updates from Google.

Prior to the Brunch process, all one needs to do is update their Chromebook's firmware (if not UEFI), using [MrChromebox's custom coreboot/UEFI firmware](https://coolstar.org/chromebook/windows-install.html#!).
1. Select Manufacturer
2. Select CPU
3. Select Model
4. Select "Developer Mode & Flash Firmware"
5. Follow instructions

(Single Boot Only) After updating firmware and running through the entire Brunch process: sudo edit-grub-config from Chrome OS terminal (shell) to silence the console at load (add "console=", and then set the timeout to 0 to silence grub entirely on boot. 

This will produce a very clean boot, without the grub interface or loading scripts showing. 

HP Pavilion 14 Chromebook (2013)
Chrome OS 86 using Brunch
Intel Celeron 847
8GB DDR3 RAM
240GB Adata SSD

**Comment by YashAgarwalDev** [comment]
great tip  if any one want to hide.

**Comment by Nunthewiser** [comment]
I made a Dell Chromebook 11 and Acer C720 better than new (Play store added) with Brunch. Just updated the Acer to r87- no issues found yet.

**Reply by uciharis** [reply]
wow that was great ! would do you spread the full tutorial ? or something video preferences to install

**Comment by kerta1n** [comment]
Just make sure that the recovery image you use doesn't have a close AUP expiry date. Remember folks, we are still tricking Google's servers that we have an actual chromebook, so no matter what image you use an AUP will still exist. Check your AUP [here](https://support.google.com/chrome/a/answer/6220366).

**Reply by nituladaboneko** [reply]
if ur not using the google recovery image and just using rammus you're ok right?
because my HP shows as aue 1998 on that link you provide
thanks

**Reply by uciharis** [reply]
would u tell me what version of the recov image u succesfully use ?

**Reply by kerta1n** [reply]
> if ur not using the google recovery image and just using rammus you're ok right?
> because my HP shows as aue 1998 on that link you provide
> thanks

Rammus expires 2026.

**Reply by kerta1n** [reply]
> would u tell me what version of the recov image u succesfully use ?

It depends, you can try hatch however

**Comment by uciharis** [comment]
i thought the tutorial above is just for windows user

**Comment by ChocolateLoverRaj** [comment]
Which board name's os did you use? Was all the hardware working even though you didn't choose the exact matching board name?
